esp/src/eclwatch/ESPLogicalFile.js

```diff
@@ -84,10 +84,15 @@ define([
             }
         },
         preProcessRow: function (item, request, query, options) {
+            var convertNull;
+            if (item.IntSize === null) {
+                convertNull = 0;
+            }
             lang.mixin(item, {
                 __hpcc_id: createID(item.NodeGroup, item.Name),
                 __hpcc_isDir: false,
-                __hpcc_displayName: item.Name
+                __hpcc_displayName: item.Name,
+                Size: convertNull //superfiles causing issues sorting by returning null
```

Comment:
Should this be IntSize?  I am not following what is happening here, if an item has an ```IntSize of null```, your setting ```Size to 0``` and if it has an ```IntSize``` your setting ```Size to Undefined``` (which lang.mixMin _might_ ignore)?